spec/services/collators/disposable_income_collator_spec.rb

```diff
@@ -69,6 +69,19 @@ module Collators
         end
       end
 
+      context "when total disposable income is negative" do
+        before do
+          assessment.gross_income_summary.update!(total_gross_income: total_outgoings - 1500.0)
+        end
+
+        it "returns the correct negative amount" do
+          collator
+          result = assessment.gross_income_summary.total_gross_income + assessment.gross_income_summary.gross_employment_income - disposable_income_summary.reload.total_outgoings_and_allowances
+          expect(disposable_income_summary.total_disposable_income).to eq result
+          expect(disposable_income_summary.total_disposable_income).to be_negative
+        end
```

Comment:
nit is it possible to break this up.

```suggestion

          let(:result) do 
            assessment.gross_income_summary.total_gross_income + 
            assessment.gross_income_summary.gross_employment_income - 
            disposable_income_summary.reload.total_outgoings_and_allowances
          end
          
          before { collator }
          
        it "returns the correct amount" do
          expect(disposable_income_summary.total_disposable_income).to eq result
        end
        
        it "can be a negative amount" do
          expect(disposable_income_summary.total_disposable_income).to be_negative
        end
```

Reply:
I have not tried this though

Reply:
@kmahern have created PR https://github.com/ministryofjustice/check-financial-eligibility/pull/861 which breaks this up if you like. but not a blocker

Reply:
Since we started following the GOVUK Rubocop style guide, they [have a note](https://github.com/alphagov/rubocop-govuk/blob/main/config/rspec.yml#L23-L26) 
``` text
# We accept multiple expectations (within reason), preferring
# them to running mulitple similar tests.
RSpec/MultipleExpectations:
  Enabled: false
```

I don't see two expectations as a problem in this instance 🤷‍♂️

Reply:
Different versions of rubocop with different rules 🤯  . I think will leave as is in that case thanks both.